src/box/iproto.cc

```diff
@@ -2872,6 +2872,48 @@ iproto_session_notify(struct session *session,
 
 /** }}} */
 
+/**
+ * Sends a packet with the given header and body over the IPROTO session's
+ * socket.
+ * On success, a packet is written to the session's output buffer, which is
+ * flushed asynchronously using Kharon.
+ *
+ * @param sid IPROTO session identifier
+ * @param header MsgPack encoded header
+ * @param header_end MsgPack encoded header end
+ * @param body MsgPack encoded body
+ * @param body_end MsgPack encoded body end
```

Comment:
I would recommend you to drop such doxygen comments from non-public API. They are not really helpful because the API is obvious anyway, and they tend to be incorrect anyway, very often from start. For example, you mention here `sid`, but there is no such param.

Reply:
Thanks for sharing your thoughts on this — dropped these comments.